tests/fixtures/display_set.json

```diff
@@ -16,9 +16,7 @@
         "enabled": false,
         "modes": {
           "time_based": {
-            "enabled": true,
-            "local_start_time": "01:00:39",
-            "start_time": "00:00:39"
+            "enabled": false
```

Comment:
I'm altering this payload to more accurately represent a screensaver set operation—since we can't currently set screensaver mode parameters, I'm removing them for now.